src/index.ts

```diff
@@ -0,0 +1,32 @@
+import * as chalk from "chalk";
+import * as tty from "tty";
+
+
+/**
+ * Prints a message indicating Ctrl+C was pressed then exits the process.
+ */
+export function defaultCtrlCHandler(): void {
+    console.log(`'${chalk.cyan("^C")}', exiting`);
+    process.exit();
```

Comment:
Actually, I'm going to use the following:

```ts
process.kill(process.pid, "SIGINT");
```

That's from "How to be a Proper Program" section on [this page](https://www.cons.org/cracauer/sigint.html).